nanshe_ipython.ipynb

```diff
@@ -1092,8 +1097,50 @@
         "if os.path.exists(data_proj_html):\n",
         "    os.remove(data_proj_html)\n",
         "\n",
-        "bp.output_file(data_proj_html, title=os.path.splitext(data_proj_html)[0])\n",
-        "bp.save(plot_group)"
+        "\n",
+        "def indent(text, spaces):\n",
+        "    spaces = \" \" * int(spaces)\n",
+        "    return \"\\n\".join(imap(lambda l: spaces + l, text.splitlines()))\n",
+        "\n",
+        "def write_html(filename, title, div, script):\n",
+        "    indent(bokeh.resources.CDN.render(), 8)\n",
```

Comment:
Hmm...this line shouldn't be here. Looks like it was left over from testing.

Removing it in PR ( https://github.com/nanshe-org/nanshe_workflow/pull/33 ).